fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidConstants.java

```diff
@@ -54,6 +59,26 @@ public static long fromBucketFraction(long numerator, long denominator) {
 		}
 	}
 
+	// ==========================
+	// ==== FLUID ATTRIBUTES ====
+	// ==========================
+	/**
+	 * Water temperature, in Kelvin.
+	 */
+	public static final int WATER_TEMPERATURE = 300;
+	/**
+	 * Lava temperature, in Kelvin.
+	 */
+	public static final int LAVA_TEMPERATURE = 1300;
+
+	public static final int WATER_VISCOSITY = 1000;
+	public static final int LAVA_VISCOSITY = 6000;
+	public static final int LAVA_VISCOSITY_NETHER = 2000;
+	/**
+	 * For flowable fluids, the viscosity should match {@code #VISCOSITY_RATIO} * {@link FlowableFluid#getFlowSpeed}.
```

Comment:
```suggestion
	 * For flowable fluids, the viscosity should match {@code VISCOSITY_RATIO} * {@link FlowableFluid#getFlowSpeed}.
```
Not a link so `#` probably shouldn't be there